sn_networking/src/bootstrap.rs

```diff
@@ -12,8 +12,8 @@ use tokio::time::Duration;
 
 use crate::target_arch::{interval, Instant, Interval};
 
-/// The interval in which kad.bootstrap is called
-pub(crate) const BOOTSTRAP_INTERVAL: Duration = Duration::from_secs(5);
+/// The interval in which kad.bootstrap is called.
```

Comment:
we no longer using kad.bootstrap anymore, instead using the `network discovery`.
the comment can now be updated.